demo/src/screens/PlaygroundScreen.js

```diff
@@ -1,38 +1,57 @@
 import React, {Component} from 'react';
```

Comment:
can you remove the PlaygroundScreen file from this PR, the PlaygroundScreen should be private and should not be committed.